test/e2e/suite_test.go

```diff
@@ -50,10 +50,48 @@ var (
 		medusaRestoreDone       pollingConfig
 		datacenterUpdating      pollingConfig
 	}
+)
 
-	logKustomizeOutput = flag.Bool("logKustomizeOutput", false, "")
-	logKubectlOutput   = flag.Bool("logKubectlOutput", false, "")
-	imageTag           = flag.String("imageTag", "latest", "")
+var (
+	configFileFlag = flag.String(
+		"configFile",
+		"./build/kind-kubeconfig",
+		"The path to the kubeconfig file to use to create remote clients and ClientConfigs. "+
+			"The default path points to the kubeconfig file generated by setup-kind-multicluster.sh. "+
+			"The path must be relative to the k8ssandra-operator repository root.",
+	)
+	controlPlaneFlag = flag.String(
+		"controlPlane",
+		"kind-k8ssandra-0",
+		"The name of a Kubernetes context to use as the control plane context.",
+	)
+	dataPlanesFlag = flag.String(
+		"dataPlanes",
+		"kind-k8ssandra-1",
+		"A comma-separated list of Kubernetes context names to use as data planes. "+
+			"The control plane should not be included here.",
+	)
+	logKustomizeOutput = flag.Bool(
+		"logKustomizeOutput",
+		false,
+		"Whether output of kustomize should be logged.",
+	)
+	logKubectlOutput = flag.Bool(
+		"logKubectlOutput",
+		false,
+		"Whether output of kubectl should be logged.",
+	)
+	imageTag = flag.String(
+		"imageTag",
+		"latest",
+		"The k8ssandra-operator image to use.",
+	)
+)
+
+var (
+	configFile   string
```

Comment:
What do you think about renaming `configFile` to `kubeconfigFile` (or something similar)?